src/extensions/updateHelper.ts

```diff
@@ -154,6 +155,25 @@ async function getConsent(field: string, message: string): Promise<void> {
   }
 }
 
+/**
+ * Displays an update warning as markdown, and prompts the user for confirmation.
+ * @param updateWarning The update warning to display and prompt for.
+ */
+export async function displayUpdateWarning(updateWarning: UpdateWarning): Promise<void> {
```

Comment:
s/display/confirm ? This should probably make it clear that an action would be taken when this function is used